docs/posts/ballerina-meets-hasura.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Ballerina Meets Hasura
+author: Anjana Fernando
+date: 15 June 2018
+status: Published
+abstract: How Hasura added Ballerina support to their platform
+socialmediaimage: codes-coding-computer-247791.jpg
+---
+
+# Ballerina Meets Hasura
+
+One of our technology partners, Hasura, has just rolled out their Ballerina integration.
+
+Hasura is a platform to build and deploy microservices-based apps easily. One of the main features of the platform is that once you have your microservices ready, a simple “git push” will deploy them to the cloud - this is the simplicity that Hasura brings in. Now pair this with Ballerina’s ability to rapidly write feature rich and resilient microservices, and this becomes a combination that makes your microservices development, testing and deployment a much more effective and seamless experience.
```

Comment:
Add a serial comma after 'testing'. Otherwise 'testing and deployment' is basically considered as a single item in the list. Apply elsewhere for any other lists in this PR.